src/bluefin_v2_client/client.py

```diff
@@ -894,7 +895,7 @@ async def get_user_leverage(self, symbol: MARKET_SYMBOLS, parentAddress: str = "
 
         for i in account_data_by_market:
             if symbol.value == i["symbol"]:
-                return fromDapiBase(int(i["selectedLeverage"]))
+                return from1e18(int(i["selectedLeverage"]))
```

Comment:
This is not required because we are getting this from Dapi which is already in 1e18

Reply:
But here before returning to user we are diving it by 1e18